config/src/main/java/com/alibaba/fescar/config/ConfigurationFactory.java

```diff
@@ -46,12 +46,14 @@
      */
     public static Configuration getInstance() {
         ConfigType configType = null;
+        String configTypeName = null;
         try {
-            configType = ConfigType.getType(
-                FILE_INSTANCE.getConfig(ConfigurationKeys.FILE_ROOT_CONFIG + ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR
-                    + ConfigurationKeys.FILE_ROOT_TYPE));
+            configTypeName = FILE_INSTANCE.getConfig(ConfigurationKeys.FILE_ROOT_CONFIG + ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR
+                    + ConfigurationKeys.FILE_ROOT_TYPE);
+            configType = ConfigType.getType(configTypeName);
         } catch (Exception exx) {
             LOGGER.error(exx.getMessage());
+            throw new NotSupportYetException("not support register type: " + configTypeName);
```

Comment:
If it throws an exception, why not remove the log here?

Reply:
i have removed the log